coax/policy_objectives/_soft_pg.py

```diff
@@ -54,4 +54,9 @@ def objective_func(self, params, state, hyperparams, rng, transition_batch, Adv)
         chex.assert_rank([W, Q], 1)
         objective = W * Q
 
+        dist_params, _ = zip(*[self.pi.function(params, state, next(rngs),
+                                                self.pi.observation_preprocessor(
+            next(rngs), s_next), True)
+            for s_next in transition_batch.extra_info['states']])
+        dist_params = (dist_params, jnp.asarray(transition_batch.extra_info['dones']))
```

Comment:
This does not yet check, whether the extra_info is even there. But I wanted to write a proof of concept before polishing the code.